src/draw/lv_draw_image.c

```diff
@@ -104,8 +104,6 @@ void lv_draw_image(lv_layer_t * layer, const lv_draw_image_dsc_t * dsc, const lv
         return;
     }
 
-    LV_PROFILER_DRAW_BEGIN;
```

Comment:
Why removed? This will result in an unpaired.

Reply:
It's in the `else` branch , not custom draw